include/SFML/Window/Window.hpp

```diff
@@ -59,6 +59,29 @@ class SFML_WINDOW_API Window : GlResource, NonCopyable
 public:
 
     ////////////////////////////////////////////////////////////
+    /// \brief Enumeration of the native system cursor types
+    ///
+    ////////////////////////////////////////////////////////////
+    enum Cursor
+    {
+        Arrow,                  ///< Arrow cursor (default)
+        ArrowWait,              ///< Busy arrow cursor
+        Wait,                   ///< Busy cursor
+        Text,                   ///< I-beam, cursor when hovering over a field allowing text entry
```

Comment:
I would rename 'Text' to 'IBeam' directly in the enum. It makes more sense IMO. (not sure if it helps, but it is the way the [.NET framework also calls it](https://msdn.microsoft.com/en-us/library/system.windows.input.cursors.ibeam%28v=vs.110%29.aspx))